.github/workflows/dart.yml

```diff
@@ -39,6 +40,15 @@ jobs:
         shell: bash
       - name: Analyze code
         run: dart analyze --fatal-infos .
+      - name: Check that grpc-web sample builds with DDC
+        if: ${{ matrix.sdk == 'dev' }}
+        run: |
+          dart pub global activate webdev
+          pushd example/grpc-web
+          rm -rf build
+          dart pub global run webdev build --no-release
```

Comment:
Consider leaving a comment that you run `--no-release` to build the DDC app (I suppose?)

Reply:
Done.